### DISCUSSION TITLE
Which CC2531 firmware versions are compatible with the latest version of zigbee2mqtt

### DISCUSSION THREAD

**Discussion by bromnl** [Koenkk/zigbee2mqtt#6894] [General]
Because of a new device I updated my zigbee2mqtt and the zigbee-herdsman-converters but after this update the service is not getting any message from my CC2531 stick.
My Stick has an old firmware version '20190608'.
Do I have to update my stick to get it working again or do I have to seach for an other problem?

I have of course saved and restored my configuration.yaml to keep the network key and connected devices

**Comment by Koenkk** [comment]
This old version is still compatible, can you provide your log? Probably your CC2531 stick crashed and reflashing should solve it.

**Reply by bromnl** [reply]
Koen,
Thanks for helping me.
If the old firmware is compatible I would like to know if the zigbee-herdsman directory should also be updated if the zigbee2mqtt and zigbee-herdsman-connectors are upgraded to the last version. I created debug log for starting and stopping the service.
[log.txt](https://github.com/Koenkk/zigbee2mqtt/files/6238652/log.txt)

**Reply by Koenkk** [reply]
How do you update zigbee2mqtt? Because zigbee-herdsman should be automatically updated.

**Reply by bromnl** [reply]
I did a git pull on zigbee2mqtt and a npm ci.
And I did make a clone on zigbee-herdsman-connectors and copied  the files to the subdirectories. Did a chown on these copied files and performed a npm ci on the zigbee-herdsman-connectors directory.

In the log file I see that a connection to the mqtt broker is made an I see messages from other devices that are connected to the mqtt broker. I do not see any messages from zigbee devices.

**Reply by Koenkk** [reply]
just a pull and npm ci is enough to update everything.

do you see anything in the z2m debug log when triggering a device? If no, try to repair it.

See https://www.zigbee2mqtt.io/information/debug.html on how to enable debug logging.

**Comment by bromnl** [comment]
I think my cc2531 is broken. I re-installed zigbee2mqtt and reflased the stick with the old an new firmware. But I never did get a message from one of the devices in the log. Even not when trying to reconnect from that device. I bought a new at aliexpress and I hope that will solve it. I will send a message in this thread when I get the new and retried that one. Thanks Koen.

Outlook voor Android<https://aka.ms/AAb9ysg> downloaden

________________________________

**Reply by bromnl** [reply]
Is there a way to be sure the cc2531 is broken?

**Reply by Koenkk** [reply]
If z2m starts I don't expect it to be broken. Try turning every device off first (remove them from power), permit joining and try pairing.

**Reply by bromnl** [reply]
That was a good tip Koen. I turned of all the powered devices an reconnected them one at the time. It work again. Thanks.👍
